invenio_records_rest/views.py

```diff
@@ -433,7 +433,13 @@ def get(self, **kwargs):
 
         # Arguments that must be added in prev/next links
         urlkwargs = dict()
-        search = self.search_class().params(version=True)
+        self.search_index = current_app.config[
```

Comment:
I not sure I understand why ``search_class`` should not include the index. Above a partial is created which should have the keyword arguments: https://github.com/dinosk/invenio-records-rest/blob/53023c55077dc43b1b643e4b460b6c529d3ffdd7/invenio_records_rest/views.py#L212